api/package.json

```diff
@@ -84,6 +84,7 @@
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "chai-sorted": "^0.2.0",
+    "deep-equal-in-any-order": "^1.0.28",
```

Comment:
on a besoin d'une lib spécifique pour faire ça ?

`expect(...).to.have.members(...)` fait pas le boutot déjà ?

Reply:
Comme indiqué dans la PR, je prends la solution la plus simple
J'ai essayé ta solution sur le test `should return target profile with learning content`=> KO
`expect(targetProfile).to.have.members(expectedTargetProfile);`